assets/css/dashboard.css

```diff
@@ -6,6 +6,13 @@ a:hover {
     color: #3291e6;
 }
 
+.go-back {
```

Comment:
If you remove your back button, this should also be removed.

Reply:
Ok

Reply:
Resolved.